app/controllers/spree/recipes/taxons_controller.rb

```diff
@@ -7,26 +7,32 @@ class TaxonsController < Spree::StoreController
       include Spree::CacheHelper
       helper 'spree/products'
 
-      def index
+       def index
+        @first_recipe = Spree::Recipe.first
```

Comment:
why we add this?

Reply:
please drop message here why we use first recipe?

Reply:
"Get Recipe" can point to any recipe for now. I will change this for staging/production. It will be static in the code.